src/test/java/fr/sncf/osrd/infra_state/RouteStateTest.java

```diff
@@ -24,16 +35,12 @@
 import fr.sncf.osrd.simulation.changelog.ArrayChangeLog;
 import fr.sncf.osrd.utils.PathUtils;
 import fr.sncf.osrd.utils.moshi.MoshiUtils;
-import org.junit.jupiter.api.Disabled;
-import org.junit.jupiter.api.Test;
-
-import java.io.IOException;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
 
 public class RouteStateTest {
-    @Test
-    public void testSimpleReserve() throws InvalidInfraException {
+    
+    @ParameterizedTest
+    @ValueSource(strings = {"NORMAL", "CBTC"})
```

Comment:
This test and the following ones could be splited in two (normal/cbtc). I think that readability should be prioritized over factoring.